crates/driver/src/tests/setup/mod.rs

```diff
@@ -72,13 +72,28 @@ pub enum Score {
     RiskAdjusted { success_probability: f64 },
 }
 
+#[serde_as]
+#[derive(Debug, Clone, PartialEq, serde::Serialize)]
+#[serde(rename_all = "camelCase", tag = "kind")]
+pub struct PriceImprovementQuote {
+    pub buy_amount: eth::U256,
+    pub sell_amount: eth::U256,
+    pub fee: eth::U256,
```

Comment:
if we define this for this test, let's also call this fee here network_fee (for clarity)